CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
+
+Features:
+
+- [Artifacts] Introduce a new `mlflow.artifacts.download_artifacts()` API that mirrors functionality of CLI `download_artifacts` (#5585, @dbczumar)
+- [Models] Add support for saving artifact if destination is an empty directory. (#5572, @akshaya-a)
+- [Models] Introduce support for `diviner` as a flavor. (#5553, @BenWilson2)
+- [Models] Add `mlflow_version` to logged model metadata. (#5515, @r3stl355)
+- [Models] Add `pyfunc.get_model_dependencies()` API to retrieve reproducible environment configuration for pyfunc flavor. (#5503, @WeichenXu123)
+- [Models] Add input example and signature autologging for `tensorflow.keras`. (#5461, @bali0019)
+- [Models] Add `code_paths` argument to all model flavors to support logging custom module code for external-dependencies for a model. (#5448, @stevenchen-db)
+- [Models / Projects] Introduce `--env-manager` configuration to prepare for support for pyenv & virtualenv. Deprecate `--no-conda` (#5567, @harupy)
+- [Scoring] Introduce `--env-manager` configuration for selection of either local or conda environment for serving. (#5487, @WeichenXu123)
+- [Scoring] Add support for `numpy.ndarray` type inputs to `tensorflow` predict function (#5545, @WeichenXu123)
+- [Scoring] Add support for Docker image deployment to Sagemaker Serverless (#5610, @matthewmayo)
+- [Tracking] Introduce a new fluent API `mlflow.last_active_run()` that will return `mlflow.active_run()` when called. (#5584, @MarkYHZhang)
+- [Tracking] Add argument `experiment_names` in the fluent `mlflow.search_runs()` API to support run search by experiment names. (#5564, @r3stl355)
+- [Tracking] Add support for specifying or overriding a run description in the fluent API `start_run()` with an argument. (#5534, @dogeplusplus)
+- [Tracking] Add `log_every_n_step` feature to `mlflow.pytorch.autolog()` to log metrics during training for PyTorch. (#5516, @adamreeve)
+- [Tracking] Add support for `pyspark.ml.param.Params` types for `Estimators`. (#5481, @serena-ruan)
+- [Tracking] Add support for tracking `pyspark.ml.Transformer` for autologging. (#5466, @serena-ruan)
+- [Tracking] Add support for custom metrics to support auto detection and loading of user-generated artifacts. (#5405, @MarkYHZhang)
+- [UI] Update UI (#5577, @harupy)
+
+Bug fixes and documentation updates:
+
+- [Artifacts] Add compatibility for reading `ENV` variables for GCS to prevent timeouts. (#5483, @sebastian-montero)
+- [Artifacts] Fix for EvaluationResult to save artifacts with appropriate file names. (#5476, @MarkYHZhang)
+- [Artifacts] Add `ENV` variables for GCS to control upload/download chunk size and timeouts (#5438, @mokrueger)
+- [Build] Fix mlflow-skinny version stack frame inspection failure (#5573, @BenWilson2)
+- [Docs] Update example for deployment to AzureML and guidance on using the officially supported Microsoft plugin. (#5491, @santiagxf)
+- [Models] Add mlflow_version to Java model spec to fix MLeap serving issue (#5576, @dbczumar)
```

Comment:
Can we include the PR for this in the feature entry for including `mlflow_version` in the MLflow Model spec? I.e. remove this line and adopt https://github.com/mlflow/mlflow/pull/5645/files#r846544456